froide/foirequest/tests/test_mail.py

```diff
@@ -296,6 +296,11 @@ def test_html_only_mail(self):
         self.assertIn(" ( https://", mail.body)
         self.assertIn("*peter.mueller@kreis-steinfurt.de*", mail.body)
 
+    def test_invalid_mail_address(self):
```

Comment:
Ideally this should be more a unit test (and less like a integration test): parsing a whole email just to test parsing of addresses makes this test not very specific. Also adding a large email with an image attachment (?!) is too much.

Reply:
Thank you for the feedback! I agree this is to unspecific. I will remove this test and do it more like this (without parsing a whole email):
```suggestion
    def test_invalid_mail_address(self):
        values = ["To: 'John Doe [#123]' <john.doe@example.com>"]
        ret = get_address_list(values)
        self.assertEqual('john.doe@example.com', ret[0].email)
```